components/information-modal.html

```diff
@@ -0,0 +1,23 @@
+<div class="flex flex-col justify-center min-h-screen pt-4 bg-gray-100 min-w-screen justify-center items-center">
+
+    <div class="rounded-lg shadow-lg bg-white my-3 max-w-sm overflow-x-auto">
+        <div class="flex border-b border-gray-100 px-5 py-4">
+            <div class="flex items-center justify-center w-5">
+                <svg class="w-6 h-6 text-white fill-current text-blue-500" viewBox="0 0 40 40"
```

Comment:
Don't think this text color will be necessary since you have the blue one overwriting it.

```suggestion
                <svg class="w-6 h-6 fill-current text-blue-500" viewBox="0 0 40 40"
```